src/utils/uploadImage.js

```diff
@@ -0,0 +1,29 @@
+import axios from 'axios';
+import { apiUrl } from '../constants';
+
+// send images in the form of bytes to the server
+const uploadImage = async (imageUri) => {
+  try {
+    const formData = new FormData();
+    formData.append('uploaded-image', {
+      uri: imageUri,
+      type: 'image/jpg',
+      name: 'image.jpg'
+    });
+    const response = await axios.post(apiUrl + '/image/upload', formData, {
```

Comment:
if we are going with https://github.com/Peteredwardj/readease-api/pull/6 , we can change this endpoint to just `/upload` 🙏

Reply:
Also, the server will return the Text object. So maybe we can handle that somewhere?

Reply:
yup sure